uvo_scraper.rb

```diff
@@ -0,0 +1,41 @@
+require 'curb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.